### PR TITLE
Config option to exclude projection PCU

### PR DESCRIPTION
**Pull request by TheCrunchy**
New config option and button that will remove projection PCU when adding up the grids total PCU.

### CODE REVIEW

File: ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs
Comment (LordTylus) [inline ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs:72]: Could you please change Brackets to match the rest of the plugin?

File: ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs
Comment (LordTylus) [inline ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs:106]: This looks like Code Duplikation to me, could you extract it into its own method so we only have that once.

Also does the BlockCount Strategy have the same issue as the PCU strategy?

File: ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs
Comment (LordTylus) [inline ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs:110]: I am not a big fan of this naming. I mean "projector" and "projectors" and "gridTerminalSystem" is not that more complicated and makes it easer to understand the method.

**Review by LordTylus — CHANGES_REQUESTED** [review]
Looks like a very nice addition to me. I would very much like if you could apply the changes requested in the comments.

And Please remove the .dll and .pdb as they are not supposed to be checked in. Actually the bin folder should be on gitignore so I am a bit puzzled on how you got them checked in. 

Ill merge it once reworks are complete.

File: ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs
Comment (TheCrunchy) [reply ALE-Biggest-Grids-Broadcast/GridDetection/BiggestGridDetectionPcuStrategy.cs]: Moved it into the CountProjectionPCU method, for the BlockCount Strategy it doesn't count projected blocks towards the topgrid.

**Review by TheCrunchy — COMMENTED** [review]
I moved the counting of projected pcu into a new method, renamed those couple variables and made the brackets match the rest.

**Review by LordTylus — APPROVED** [review]
Code-Style is not necessarily to my liking, but this I will do myself.

Only one question needs to be answered still before merge. But since I assume you tested it thoroughly it has my approval :-)

**Pull request merged**
